src/tree/localProject/ConfigGroupTreeItem.ts

```diff
@@ -6,16 +6,18 @@
 import { AzExtParentTreeItem, AzExtTreeItem, TreeItemIconPath } from "vscode-azureextensionui";
 import { localize } from "../../utils/localize";
 import { treeUtils } from "../../utils/treeUtils";
+import { EnvironmentTreeItem } from "../EnvironmentTreeItem";
 import { ConfigTreeItem } from "./ConfigTreeItem";
-import { LocalProjectTreeItem } from "./LocalProjectTreeItem";
+
+export type BuildConfig = 'app_location' | 'api_location' | 'output_location';
 
 export class ConfigGroupTreeItem extends AzExtParentTreeItem {
     public static contextValue: string = 'azureStaticConfigGroup';
     public contextValue: string = ConfigGroupTreeItem.contextValue;
     public readonly label: string = localize('config', 'Configuration');
```

Comment:
I think we should call this GitHub Configurations especially since it's on the remote SWA.